### PR TITLE
✨ Shift left the labeling of edge CRDs for kube-bind to export

### PR DESCRIPTION
**Pull request by waltforme**
## Summary
This PR uses some more kubebuilder markers to label the edge CRDs earlier for them to be later exported via kube-bind.

### CODE REVIEW

File: config/crds/edge.kubestellar.io_customizers.yaml [hunk 1]
```diff
@@ -3,8 +3,9 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
-  creationTimestamp: null
```
Comment (MikeSpreitzer) [inline config/crds/edge.kubestellar.io_customizers.yaml:7]: Why was this line here (and in the other YAMLs)? Does removing it break something?

Comment (waltforme) [reply config/crds/edge.kubestellar.io_customizers.yaml:7]: As far as what I saw in local test and the CI, I didn't see it break anything. I think it is introduced by the more recent version of kubebuilder.

Comment (waltforme) [reply config/crds/edge.kubestellar.io_customizers.yaml:7]: Yes
